app/assets/src/components/BulkUploadImport.jsx

```diff
@@ -271,7 +272,7 @@ class BulkUploadImport extends React.Component {
         that.setState({
           submitting: false
         });
-        that.gotoPage(`/?ids=${that.state.createdSampleIds.join(',')}`);
+        that.gotoPage(`/?project_id=${that.state.projectId}`);
       }, 2000)
```

Comment:
was the problem perhaps that this timeout is only 2000 milliseconds, i.e., enough to submit a single sample but not enough to submit a batch of samples?